### PR TITLE
Fixed #33030 -- Fixed broken links to GDAL docs.

### PR DESCRIPTION
**Pull request by MaertHaekkinen**
https://code.djangoproject.com/ticket/33030

### CODE REVIEW

File: docs/ref/contrib/gis/gdal.txt [hunk 1]
```diff
@@ -1873,7 +1873,7 @@ For commercial storage providers (e.g. ``/vsis3/``) the system should be
 previously configured for authentication and possibly other settings (see the
 `GDAL Virtual Filesystems documentation`_ for available options).
 
-.. _`GDAL Virtual Filesystems documentation`: https://gdal.org/user/virtual_file_systems.html
```
Comment (felixxm) [inline docs/ref/contrib/gis/gdal.txt:1876]: The current links (with `.html`) work so I see no reason to change them. Moreover `gdal.org` uses them. Please revert changes in `docs`.

Comment (felixxm) [reply docs/ref/contrib/gis/gdal.txt:1876]: I would also add `.html` suffixes to links updated in the source code.

**Review by felixxm — COMMENTED** [review]
@MaertHaekkinen Thanks for this patch :+1:

File: docs/ref/contrib/gis/gdal.txt (hunk 1 above)
Comment (MaertHaekkinen) [reply docs/ref/contrib/gis/gdal.txt]: Got it, I've applied these changes

**Comment by felixxm** [comment]
@MaertHaekkinen Thanks for updates :+1: Welcome aboard :boat:

**Pull request merged**